src/main/java/ch/iceguard/functions/Function.java

```diff
@@ -1,77 +1,60 @@
 package ch.iceguard.functions;
 
-import java.util.*;
-import java.util.function.Consumer;
-
-import com.microsoft.azure.functions.annotation.*;
 import com.microsoft.azure.functions.*;
-import com.mongodb.MongoClient;
-import com.mongodb.MongoClientURI;
-import com.mongodb.client.MongoCollection;
-import com.mongodb.client.MongoDatabase;
-import org.bson.Document;
+import com.microsoft.azure.functions.annotation.AuthorizationLevel;
+import com.microsoft.azure.functions.annotation.FunctionName;
+import com.microsoft.azure.functions.annotation.HttpTrigger;
 
-import static com.mongodb.client.model.Filters.eq;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
 
 /**
  * Azure Functions for getting Data from Cosmos DB with Mongo Api.
  */
 public class Function {
 
-    private static final String MONGO_DB_CONNECTION_STRING_PROPERTY = "iceguard-iot_DOCUMENTDB";
-    private static final String MONGO_DB_COLLECTION_NAME_PROPERTY = "collection-name-COSMOS";
-    private static final String MONGO_DB_NAME = "iceguard-iot";
-    private static final String FIELD_DEVICE_ID = "deviceId";
+    static final String FIELD_DEVICE_ID = "deviceId";
+    private static final String FIELD_DEVICE_IDS = "deviceIds";
+    private static final int MAX_RESULTS = 50;
```

Comment:
Why 50?

Reply:
I think it's a good value to start with. Later this should be parametrized so that the request can decide on how many measurements it should get. If this will be done, we should also think about pagination.